en/docs/chapter_tree/summary.md

```diff
@@ -3,24 +3,24 @@
-- Related terminology of binary trees includes root node, leaf node, level, degree, edge, height, and depth, among others.
+- For a node in a binary tree, its left (right) child node and the tree formed below it is called the node's left (right) subtree.
+- Terms related to binary trees include root node, leaf node, level, degree, edge, height, and depth.
 - The operations of initializing a binary tree, inserting nodes, and removing nodes are similar to those of linked list operations.
 - Common types of binary trees include perfect binary trees, complete binary trees, full binary trees, and balanced binary trees. The perfect binary tree represents the ideal state, while the linked list is the worst state after degradation.
 - A binary tree can be represented using an array by arranging the node values and empty slots in a level-order traversal sequence and implementing pointers based on the index mapping relationship between parent nodes and child nodes.
 - The level-order traversal of a binary tree is a breadth-first search method, which reflects a layer-by-layer traversal manner of "expanding circle by circle." It is usually implemented using a queue.
 - Pre-order, in-order, and post-order traversals are all depth-first search methods, reflecting the traversal manner of "going to the end first, then backtracking to continue." They are usually implemented using recursion.
 - A binary search tree is an efficient data structure for element searching, with the time complexity of search, insert, and remove operations all being $O(\log n)$. When a binary search tree degrades into a linked list, these time complexities deteriorate to $O(n)$.
 - An AVL tree, also known as a balanced binary search tree, ensures that the tree remains balanced after continuous node insertions and removals through rotation operations.
-- Rotation operations in an AVL tree include right rotation, left rotation, right-then-left rotation, and left-then-right rotation. After inserting or removing nodes, an AVL tree performs rotation operations from bottom to top to rebalance the tree.
+- Rotation operations in an AVL tree include right rotation, left rotation, right-left rotation, and left-right rotation. After inserting or removing nodes, the AVL tree performs rotation operations from bottom to top to rebalance the tree.
 
 ### Q & A
 
 **Q**: For a binary tree with only one node, are both the height of the tree and the depth of the root node $0$?
 
 Yes, because height and depth are typically defined as "the number of edges passed."
 
-**Q**: The insertion and removal in a binary tree are generally completed by a set of operations. What does "a set of operations" refer to here? Can it be understood as the release of resources of the child nodes?
+**Q**: The insertion and removal in a binary tree are generally accomplished by a set of operations. What does "a set of operations" refer to here? Can it be understood as the release of resources of the child nodes?
```

Comment:
Can it be understood...? => Does it imply releasing the resources of the child nodes?